lib/bounces.js

```diff
@@ -108,6 +108,24 @@ module.exports = function (log, error) {
           }
         }
 
+        // Log flow metrics if `flowId` and `flowBeginTime` specified in headers
+        const flowId = getHeaderValue('X-Flow-Id', message)
+        const flowBeginTime = getHeaderValue('X-Flow-Begin-Time', message)
+        if (flowId && flowBeginTime) {
+          const eventName = 'email.' + templateName + '.bounced'
+          const currentTime = Date.now()
+
+          // Matches structure emitted by https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/context.js#L122
+          const flowEventInfo = {
+            event: eventName,
+            time: currentTime,
+            flow_id: flowId,
+            flow_time: (currentTime <= flowBeginTime) ? 0 : (currentTime - flowBeginTime)
+          }
+
+          log.info('flowEvent', flowEventInfo)
```

Comment:
Indeed we will.

@vbudhram, when it's time, you probably want to chat to @whd about this in IRC, and/or raise a bug with product set to `Cloud Services` and component set to `Metrics: Pipeline`. See, for example, [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1289474) as an example of something similar-ish.